cypress/e2e/ModelizerPage/SwitchView/delete.feature

```diff
@@ -0,0 +1,228 @@
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-label-leto-modelizer.config.json\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "default.plugin-test.js"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "leto-modelizer.config.json"
+    And  I wait 1 second
+
+    When I click on "[data-cy=\"file-tabs-container\"] [data-cy=\"file-label-default.plugin-test.js\"]"
+    Then I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "default.plugin-test.js"
+
+    When I hover "[data-cy=\"file-explorer\"] [data-cy=\"file-explorer-buttons-default.plugin-test.js\"]" to make it visible
+    And  I click on "[data-cy=\"file-explorer\"] [data-cy=\"file-explorer-buttons-default.plugin-test.js\"]"
+    Then I expect "[data-cy=\"file-explorer-action-menu\"]" exists
+
+    When I click on "[data-cy=\"file-explorer-menu-delete-file\"]"
+    Then I expect "[data-cy=\"delete-file-dialog\"]" exists
+
+    When I click on "[data-cy=\"delete-file-form\"] [data-cy=\"delete-file-submit\"]"
+    Then I expect "positive" toast to appear with text "File is deleted."
+    And  I expect "[data-cy=\"delete-file-form\"]" is closed
+    And  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" not exists
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Model"
+    And  I expect "[data-cy=\"modelizer-model-view-draw-root\"]" exists
+    And  I expect "[id^=\"truck_\"]" not exists
+
+  Scenario: Delete a link between two components (Model view) should remove corresponding attribute in object inside plugin file content (Text view)
+    Given I click on "[data-cy=\"component-definition-paper\"]"
+    And  I click on "[data-cy=\"component-definition-image\"]"
+
+    When I click on "[id^=\"paper_\"]"
+    And  I click on "[id=\"create-link\"]"
+    And  I click on "[id^=\"image_\"]"
+    Then I expect ".link" exists
+    
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-label-leto-modelizer.config.json\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "default.plugin-test.js"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "leto-modelizer.config.json"
+    And  I wait 1 second
```

Comment:
and -> when
put it in the next paragraph

Reply:
done